### ISSUE TITLE
[BUG]Upgrade to version 20.0 of whatsapp API

### ISSUE THREAD

**Issue by lucasjamar** [filipporomani/whatsapp-python#43]
Hi Filippo,
Could you please upgrade the API version to 20.0? It doesn't send messages for me anymore with 18.0.
Furthermore, how could we automate the process to also have the latest whatsapp api version?
Thanks for the repo btw!

**Comment by lucasjamar** [comment]
Actually v21.0 is the latest

**Comment by filipporomani** [comment]
Hello! I'm updating the version to 21.0, I'll push the change in a few minutes. About the automatic process, I'll give a look and see how that can be achieved! Thanks for your help!

**Comment by filipporomani** [comment]
Sorry, I completely forgot to update you. 
If you want you can upgrade the version by installing the git development one:
```
git clone https://github.com/filipporomani/whatsapp
cd whatsapp
pip install .
```
I'm trying to find a way to keep the API version always up to date, but I can't find any official way to do so and reading the html file could lead to errors (i tried it, but it's not so reliable imo). 
I'll release v4.0.0 soon, and i'll include the api version updater if I fin a way to do so!

**Comment by filipporomani** [comment]
This was implemented in the last commit! I'm releasing v4.0.0 rn

**Comment by lucasjamar** [comment]
Awesome. Many thanks
